frontend/src/routes/Home/Overview/OverviewPage.tsx

```diff
@@ -541,97 +538,137 @@ export default function OverviewPage() {
             }
         >
             <AcmScrollable>
+                {searchError && (
+                    <PageSection>
+                        <AcmAlert
+                            noClose
+                            isInline
+                            variant={searchError?.graphQLErrors[0]?.message.includes('not enabled') ? 'info' : 'danger'}
```

Comment:
I'd make this message always `info`.
Maybe we could do warning, but let's stay away from `danger`

Reply:
I'm trying to avoid `danger` because from the Overview page users are interpreting it as RCHAM (the entire product) is broken.  Now that we are able to display partial data, I want to send the message that only search (single component of RHACM) is experiencing problems.  This problem reduces functionality, but it isn't catastrophic or critical to the rest of RHACM.

Reply:
Maybe warning is a good compromise?

Reply:
Makes sense - just pushed the change.